src/rendering/renderers/gpu/WebGPURenderer.ts

```diff
@@ -24,12 +24,14 @@ import type { SystemConstructor } from '../shared/system/System';
 import type { ExtractRendererOptions, ExtractSystemTypes } from '../shared/system/utils/typeUtils';
 
 const DefaultWebGPUSystems = [
+    // we want renderStart to be called first..
+    // before the renderTarget system renderStart..
+    GpuEncoderSystem,
```

Comment:
we should use `priority` on the extension to manage this?

Reply:
yes - follow up ticket [here](https://github.com/orgs/pixijs/projects/2/views/4?pane=issue&itemId=44862617)